maestro-ios-driver/src/main/kotlin/xcuitest/api/NetworkErrorHandler.kt

```diff
@@ -21,7 +21,7 @@ class NetworkErrorHandler(
     companion object {
         const val RETRY_RESPONSE_CODE = 503
         private const val NO_RETRY_RESPONSE_CODE = 502
-        private const val MAX_RETRY = 3
+        private const val MAX_RETRY = 5
```

Comment:
I'm holding off making this configurable right away, since I noticed that 4 is enough

Reply:
just out of curiousity, how were you able to reproduce this, also using MockWebServer?

Reply:
This I noticed in production that 4th call was successful hence increasing retries